lib/monkey/deps/flb_libco/settings.h

```diff
@@ -42,6 +42,8 @@
   #define text_section __declspec(allocate(".text"))
 #elif defined(__APPLE__) && defined(__MACH__)
   #define text_section __attribute__((section("__TEXT,__text")))
+#elif defined(__clang__)
+  #define text_section __attribute__((section(".text")))
```

Comment:
Can you extract this changeset into a separate patch, and submit
it to https://github.com/edsiper/flb_libco?

Our standard workflow for a library fix is to pull the patch in the upstream repo
and bring it to the downstream. So we prefer getting this patch there.